client/my-sites/email/email-management/gsuite-users-card/index.jsx

```diff
@@ -93,7 +95,11 @@ class GSuiteUsersCard extends React.Component {
 
 	renderDomain( domain, users ) {
 		if ( hasTitanMailWithUs( domain ) ) {
-			return <TitanControlPanelLoginCard domain={ domain } key={ `titan-${ domain.name }` } />;
+			const domainKey = `titan-${ domain.name }`;
+			if ( isEnabled( 'titan/phase-2' ) ) {
+				return <TitanManagementNav domain={ domain } key={ domainKey } />;
+			}
+			return <TitanControlPanelLoginCard domain={ domain } key={ domainKey } />;
```

Comment:
Also worth keeping in mind that this will be rendered both in the domain specific view, as well as the site specific view where we list all domains that have email. Maybe we want to have a check in place to see if we're listing all domains or just one?

Same goes for the `TitanControlPanelLoginCard` component now. If we enable the iframe flag, and we have 2 domains with Titan, we render both iframes if we go to `/email/<site_slug>`, which is not ideal.

Reply:
The `TitanControlPanelLoginCard` component with the iframe has this problem already when it displays the iframe. I'm happy to just replace that with the dedicated nav moving forward.

RE showing multiple email management for a site -- does the nav take up too much space? I tried to keep it relatively small, as I noticed the logic supports showing details for multiple domains.

Reply:
> does the nav take up too much space?

I don't think so, it should be fine IMO.

Reply:
Also, just to be clear, once we flip the `titan/phase-2` feature flag, the new page that's supposed to show the embedded control panel is only visible when a specific domain is selected.